Myanmar/HaveFun.txt

```diff
@@ -0,0 +1,15 @@
+I was trying to type in Myanmar. However, the text editor I was using doesn't seem to support Myanmar Font.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.